sample/src/main/java/com/airwallex/paymentacceptance/PaymentCartFragment.kt

```diff
@@ -157,7 +157,7 @@ class PaymentCartFragment : Fragment() {
                     googlePayOptions = GooglePayOptions(
                         billingAddressRequired = true,
                         billingAddressParameters = BillingAddressParameters(BillingAddressParameters.Format.FULL),
-                        merchantId = Settings.accountId
+                        merchantId = "423102959764541877"
```

Comment:
Is this change intentional?

Reply:
yeah, it's a unique googlepay merchant ID for the demo app so that the real payment can happen to our [Airwallex account](https://pay.google.com/business/console/info/BCR2DN4TWD5IRR2E).

Reply:
Should we maybe put that in the gradle.properties file? So users can change if they need to